rel/overlays/bin/migrate

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-cd -P -- "$(dirname -- "$0")/.."
```

Comment:
I think you are breaking relative paths here.

Why the change?

Reply:
It was auto generated by `mix phx.gen.release --docker`